### ISSUE TITLE
Failing installation for iOS with cordova 9.0.0/cordova-ios 5.0.0

### ISSUE THREAD

**Issue by zarko-tg** [ReallySmallSoftware/cordova-plugin-firebase-crashlytics#16]
## Expected Behavior

Plugin installs correctly.

## Actual Behavior

It doesn't.

```
Installing "cordova-plugin-firebase-crashlytics" for ios
cordova-ios version check failed ("/app/platforms/ios/cordova/version"), continuing anyways.
"framework" tag with type "podspec" is deprecated and will be removed. Please use the "podspec" tag.
Running command: pod install --verbose
Failed to install 'cordova-plugin-firebase-crashlytics': Error: pod: Command failed with exit code 1
    at ChildProcess.whenDone (/app/node_modules/cordova-common/src/superspawn.js:125:23)
    at ChildProcess.emit (events.js:189:13)
    at maybeClose (internal/child_process.js:970:16)
    at Socket.stream.socket.on (internal/child_process.js:389:11)
    at Socket.emit (events.js:189:13)
    at Pipe._handle.close (net.js:597:12)
Failed to restore plugin "cordova-plugin-firebase-crashlytics" from config.xml. You might need to try adding it again.Error: Error: pod: Command failed with exit code 1
```

**Comment by zarko-tg** [comment]
In case this plugin has not be abandoned, you might want to see my attempt to have it working with cordova 9 and cordova-ios 5
https://github.com/timegrip/cordova-plugin-firebase-crashlytics/tree/cordova-ios-5.0.0

At least it installs and build now but I am not seeing that it's working in the Firebase Crashlytics console even after sending crashes and exceptions. It's like the connection has not been established at all. Android is fine, I'm getting reports and all.

**Comment by joao-jr** [comment]
In my case I get following message:

```
Failed to install 'cordova-plugin-firebase-crashlytics': CordovaError: Using "requireCordovaModule" to load non-cordova module "xcode" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
    at Context.requireCordovaModule (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/Context.js:57:15)
    at Object.removeShellScriptBuildPhase (/Users/developer/Documents/projetos/linha-direta-ionic/plugins/cordova-plugin-firebase-crashlytics/scripts/lib/ios-helper.js:78:29)
    at module.exports (/Users/developer/Documents/projetos/linha-direta-ionic/plugins/cordova-plugin-firebase-crashlytics/scripts/after_plugin_add.js:15:19)
    at runScriptViaModuleLoader (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:181:32)
    at runScript (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:157:16)
    at /usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:125:20
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
Using "requireCordovaModule" to load non-cordova module "xcode" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
[ERROR] An error occurred while running subprocess cordova.
```

**Comment by ReallySmallSoftware** [comment]
I will be picking this up in the not too distant future when I upgrade an app I have to ios-5. Many thanks for the update.

**Comment by piotr-cz** [comment]
This may be fixed #15, please check

**Comment by ReallySmallSoftware** [comment]
I have published a new release as it was being asked for, but I haven't had time to test properly
